charts/langgraph-dataplane/templates/operator/rbac.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if and .Values.operator.enabled .Values.operator.rbac.create -}}
-{{- if .Values.operator.watchNamespaces }}
-{{- $watchNamespacesList := splitList "," .Values.operator.watchNamespaces }}
+{{- if or .Values.config.watchNamespaces .Values.operator.watchNamespaces }}
+{{- $watchNamespacesList := splitList "," (default .Values.operator.watchNamespaces .Values.config.watchNamespaces) }}
```

Comment:
For backward compatibility, default to .Values.operator.watchNamespaces if .Values.config.watchNamespaces is not set.